yolo.js

```diff
@@ -0,0 +1,92 @@
+var model = [
+    {
+        name: "cat 1",
+        imageUrl: "https://images.pexels.com/photos/104827/cat-pet-animal-domestic-104827.jpeg?auto=compress&cs=tinysrgb&h=350",
+        clickCount: 0,
+        id: 0
+    },
+    {
+        name: "cat 2",
+        imageUrl: "https://images.pexels.com/photos/20787/pexels-photo.jpg?auto=compress&cs=tinysrgb&h=350",
+        clickCount: 0,
+        id: 1
+    }
+];
+
+var controller = {
+    init: function(){
+        view.renderList(model)
+        view.renderCat(0)
+    },
+    countAddOne: function(catId){
+        model[catId]["clickCount"] ++
+        view.updateCount(catId)
+    },
+    updateCat: function(catId){
```

Comment:
Podobna zgodba kot zgoraj:
Podaj vrednosti sprememb v funkcijo. Ne jih iskat v funkciji.
i.e.
`updateCate(catId, name, imageUrl) { model[catId].name = name }`
(ne pozabi na preverjanje ce ta macek sploh obstaja!)

Reply:
jQuery objecti niso poceni. V bistvu kakrsnakol DOM manipulacija ni poceni. 
Skratka, ce imas jQuery object e.g. `$(#newName)` ki se klice/rabi veckrat, ga zapeci v variablo. 
`var nameInput = $("#newName");`
(jQuery navada je bila, da se variable, ki kazejo na jQuery object zacnejo z $ - to je samo dobra praksa, ni pravilo)
`var $nameInput = $("#newName");`